public_domain_review/utils.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup
+
+SOURCE_NAME = "public-domain-review"
+
+
+def get_outbound_links(url, html):
+    soup = BeautifulSoup(html, "html.parser")
```

Comment:
Other functions expect a soup object but some of them expect the raw html like this one does. We should have them all take soups.